test/test_sparse_csr.py

```diff
@@ -0,0 +1,202 @@
+import torch
+
+torch.set_default_dtype(torch.double)
+
+import functools
+import random
+import operator
+import numpy as np
+import warnings
+from torch.testing._internal.common_utils import TestCase, run_tests, load_tests
+
+# load_tests from torch.testing._internal.common_utils is used to automatically filter tests for
+# sharding on sandcastle. This line silences flake warnings
+load_tests = load_tests
+
+class TestSparseCSR(TestCase):
```

Comment:
This new test suite is written in the old style, but I know @aocsa has a modernization of the existing COO sparse test suite, is there any way to go straight to the modernized version here?